docs/05-infrastructure-nodes.md

```diff
@@ -437,11 +446,12 @@ ip-10-0-217-226.ec2.internal   Ready     infra,worker   17h       v1.11.0+406fc8
 
```

Comment:
the version should be `v1.12.4+4dd65df23d` instead of `v1.11.0+406fc897d8` in above line.

Reply:
fixed with 84fb55d